### ISSUE TITLE
Add coverage of the new TinkerPop 3.3.1 math() step

### ISSUE THREAD

**Issue by krlawrence** [krlawrence/graph#21]
TinkerPop 3.3.1 introduces the math() step that adds some nice capabilities. Need to add some coverage.

**Comment by krlawrence** [comment]
Examples:
```
g.V().has('code','AUS').as('a').out('route').has('code','DFW').as('b').math('a+b').by('runways').by('runways')
==>9.0
```

**Comment by krlawrence** [comment]
http://tinkerpop.apache.org/docs/current/reference/#math-step

**Comment by krlawrence** [comment]
Fixed in the v278 (TP 3.3.1) release.
